static/js/reducers/order_receipt_test.js

```diff
@@ -9,9 +9,12 @@ import {
   setTimeoutActive,
 } from '../actions/order_receipt';
 import { createAssertReducerResultState } from '../util/test_utils';
+import type { OrderReceiptState } from './order_receipt';
 
 describe('order receipt reducer', () => {
-  let sandbox, store, assertReducerResultState;
+  let sandbox, store, assertReducerResultState: (
+    a: () => Action, f: ((s: any) => OrderReceiptState), d: any
```

Comment:
could you declare this as a parameterized type, say `AssertReducerResultState`, which would take a record type as an argument? (like `AssertReducerResultState<OrderReceiptState>`)

that would let you re-use the type declaration down below.